tests/compiler/std/math.release.wat

```diff
@@ -49900,7 +49900,7 @@
    call $~lib/builtins/abort
    unreachable
   end
-  f64.const -nan:0x8000000000000
+  f64.const nan:0x8000000000000
```

Comment:
Change in this file should be reverted. I guess you use mac arm64 as develop machine. I have the same problem.

Reply:
Ok